sinopia_client/README.md

```diff
@@ -125,19 +122,19 @@ All URIs are relative to *https://localhost*
 
 Class | Method | HTTP request | Description
```

Comment:
General question, can the documentation be generated without re-generating the code base?

Reply:
assuming i'm understanding this correctly, i don't think that's possible, at least with the `swagger-codegen` tool used here.  i'm pretty sure the documentation is generated from the swagger spec and not the JS code, so i don't think you'd be able to generate these docs from arbitrary code, if that's what you mean.

maybe what's desired in this case, for the non-swagger-specified JS code, is something like YARD/rubydoc/javadoc?  is there a JS equivalent for that?  that'd require keeping method level comments up to date, but that at least lives with the code, and so seems less onerous to update and generate off of than hand-writing docs.